python_modules/libraries/dagster-snowflake/dagster_snowflake_tests/test_resources.py

```diff
@@ -292,7 +292,7 @@ def test_fetch_last_updated_timestamps_empty():
         )
 
 
-@pytest.mark.skipif(not IS_BUILDKITE, reason="Requires access to the BUILDKITE snowflake DB")
+# @pytest.mark.skipif(not IS_BUILDKITE, reason="Requires access to the BUILDKITE snowflake DB")
```

Comment:
why commented?

Reply:
commented to run the test locally. Will get rid of before landing.